tfx/components/transform/executor_test.py

```diff
@@ -72,24 +90,35 @@ def _get_output_data_dir(self, sub_dir=None):
 
   def _make_base_do_params(self, source_data_dir, output_data_dir):
     # Create input dict.
-    examples = standard_artifacts.Examples()
-    examples.uri = os.path.join(source_data_dir, 'csv_example_gen')
-    examples.split_names = artifact_utils.encode_split_names(['train', 'eval'])
+    e1 = standard_artifacts.Examples()
+    e1.uri = self._artifact1_uri
+    e1.split_names = artifact_utils.encode_split_names(['train', 'eval'])
+    e2 = copy.deepcopy(e1)
+    e2.uri = self._artifact2_uri
+
+    self._single_artifact = [e1]
+    self._multiple_artifacts = [e1, e2]
+
     schema_artifact = standard_artifacts.Schema()
     schema_artifact.uri = os.path.join(source_data_dir, 'schema_gen')
 
     self._input_dict = {
-        executor.EXAMPLES_KEY: [examples],
+        executor.EXAMPLES_KEY: self._single_artifact,
         executor.SCHEMA_KEY: [schema_artifact],
     }
 
     # Create output dict.
     self._transformed_output = standard_artifacts.TransformGraph()
     self._transformed_output.uri = os.path.join(output_data_dir,
                                                 'transformed_graph')
-    self._transformed_examples = standard_artifacts.Examples()
-    self._transformed_examples.uri = os.path.join(output_data_dir,
-                                                  'transformed_examples')
+    te1 = standard_artifacts.Examples()
```

Comment:
Let's avoid these types of abbreviations for variables (transformed_1 for example is fine)